.gitignore

```
@@ -0,0 +1 @@
+.idea
```

It's a common convention to have an empty line at the end of each file.

```suggestion
.idea

```